### ISSUE TITLE
[Feature request] [Android] [UI Enhancement] Ask user to use xmb Menu Driver on first launch

### ISSUE THREAD

**Issue by molul** [libretro/RetroArch#4937]
Thanks to the libretro forums I discovered the xmb interface is available on Android. Much cooler than the default one! But it's a shame that such a beautiful interface is somehow hidden to newbies :)

I'd set it by default. However, I understand that there would be a problem: if the user is not using a physical controller, you can do nothing on the interface (you'd need the gamepad overlay displaying on menus, which is set to OFF by default).

Therefore, an explanation would be needed. Here's how I'd do it:

-When the user starts Retroarch for the first time, it loads the xmb Menu Driver. 

-If no physical controller is detected, I'd show a message explaining that either a physical or the virtual gamepad is needed to use the xmb interface, and then I'd ask the user to either connect a physical controller, enable the virtual one, or go to the current default Menu Driver.

Hope it helps! :)

**Comment by jvook** [comment]
I disagree.  Most android users expect a touch interface.  And even if you have a physical gamepad (I do), sometimes you don't have it with you.  Then you'd have to go into retroarch.cfg to fix it.

**Comment by molul** [comment]
"Most android users expect a touch interface"
Do you mean touch controls for the menu? Or the virtual gamepad overlay?

If it's the first one, honestly, playing most games with touch buttons is a pain. The only games I can say I enjoyed with touch controls, would be Sonic 1, 2 and CD remastered for Android. But Sonic can be controlled with a D-pad and one button. Play Street Fighter II with touch controls XDD

**Comment by jvook** [comment]
I mean that Android users expect a touch interface to the menu.

**Comment by molul** [comment]
That is, being able to select options in the menu by swiping and touching? Or having the virtual gamepad overlay always on?
